Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.11
+
+COPY ./ /www/
+
+ENV USER abc
```

Comment:
Nit: Could you please rename the user/group to `darkhttpd`?

Reply:
+1 That would be great!

Reply:
done